torch/_decomp/decompositions.py

```diff
@@ -744,23 +744,38 @@ def slice_forward(
         raise RuntimeError("slice step must be positive")
 
     start_val = start if start is not None else 0
-    end_val = end if end is not None else sys.maxsize  # 2^63 – 1
```

Comment:
Yes it would. In fact, I have broken these passes anyway, as narrows are showing up as slice_strict now not slice. Hmm, I wonder why they don't check for size of tensor instead.